### PR TITLE
IAC Clothing

### PR DESCRIPTION
**Pull request by dragoslava**
"Added a variety of IAC related sprites and made into items."
"Adds IAC to loadout with medical restrictions."

dreamseeker_2018-10-10_21-06-36

### CODE REVIEW

**Comment by dragoslava** [comment]
!review

**Comment by BotBOREALIS** [comment]
Icons with diff:
<details><summary>icons/mob/head.dmi</summary>

Key | Old | New | Status
--- | --- | --- | ---
iachat||iachat|Created
iachat[1]||![iachat[1]](http://images.aurorastation.org/images/4837006c83d994696673e5df7739c786.png)|Created
iachat[2]||![iachat[2]](http://images.aurorastation.org/images/30f284270dfe3cf46af7013e595602a1.png)|Created
iachat[3]||![iachat[3]](http://images.aurorastation.org/images/742222cb57b3208ba66cdc4aef9015f7.png)|Created
iac_helmet||iac_helmet|Created
iac_helmet[1]||![iac_helmet[1]](http://images.aurorastation.org/images/3290bbd29d8893d4a54dc286769dc206.png)|Created
iac_helmet[2]||![iac_helmet[2]](http://images.aurorastation.org/images/dc67c4a9ef272c97f9e014b6812fb926.png)|Created
iac_helmet[3]||![iac_helmet[3]](http://images.aurorastation.org/images/2865d079fa4ceeecb6c1d0e0e8d27d2c.png)|Created
iachat_flipped||iachat_flipped|Created
iachat_flipped[1]||![iachat_flipped[1]](http://images.aurorastation.org/images/3abe1aef374883739557171b5f93cc6d.png)|Created
iachat_flipped[2]||![iachat_flipped[2]](http://images.aurorastation.org/images/13e257330bf81d9d3df9649ec404df86.png)|Created
iachat_flipped[3]||![iachat_flipped[3]](http://images.aurorastation.org/images/e6170520c173b96a52f488f966639e80.png)|Created
</details>
<details><summary>icons/mob/mask.dmi</summary>

Key | Old | New | Status
--- | --- | --- | ---
iacmask||iacmask|Created
iacmask[1]||![iacmask[1]](http://images.aurorastation.org/images/2f5e4f579da183cf9c857b12e9c8a714.png)|Created
iacmask[2]||![iacmask[2]](http://images.aurorastation.org/images/9aa918c29c16c1f109006417f71ffc5b.png)|Created
iacmask[3]||![iacmask[3]](http://images.aurorastation.org/images/d302c85ae0632cf86f96c83cb575608a.png)|Created
iacmask_r||iacmask_r|Created
iacmask_r[1]||![iacmask_r[1]](http://images.aurorastation.org/images/420d264bf31a8efd23f03b2a8cd91df4.png)|Created
iacmask_r[2]||![iacmask_r[2]](http://images.aurorastation.org/images/a90eb39b352ed460340e4aa35c23f0a4.png)|Created
iacmask_r[3]||![iacmask_r[3]](http://images.aurorastation.org/images/a90eb39b352ed460340e4aa35c23f0a4.png)|Created
</details>
<details><summary>icons/mob/suit.dmi</summary>

Key | Old | New | Status
--- | --- | --- | ---
iac_vest_s||iac_vest_s|Created
iac_vest_s[1]||![iac_vest_s[1]](http://images.aurorastation.org/images/fb5d739b1f145d453fe03e5867280cf5.png)|Created
iac_vest_s[2]||![iac_vest_s[2]](http://images.aurorastation.org/images/73eef69a2724540cf3de5a9b98bb6f16.png)|Created
iac_vest_s[3]||![iac_vest_s[3]](http://images.aurorastation.org/images/87b87fa1705f1e7235576d3c4763f0a3.png)|Created
</details>
<details><summary>icons/mob/uniform.dmi</summary>

Key | Old | New | Status
--- | --- | --- | ---
iacuniform_r_s||iacuniform_r_s|Created
iacuniform_r_s[1]||![iacuniform_r_s[1]](http://images.aurorastation.org/images/5827d85924397d740b96014af2aaec6a.png)|Created
iacuniform_r_s[2]||![iacuniform_r_s[2]](http://images.aurorastation.org/images/2a4c86e0abc62ed824ab42342b178512.png)|Created
iacuniform_r_s[3]||![iacuniform_r_s[3]](http://images.aurorastation.org/images/2554f5f19781f24598a1619e110f9296.png)|Created
iacuniform_s||iacuniform_s|Created
iacuniform_s[1]||![iacuniform_s[1]](http://images.aurorastation.org/images/d67d94ef85d960c9d4693beb2e978055.png)|Created
iacuniform_s[2]||![iacuniform_s[2]](http://images.aurorastation.org/images/38f147be9bfdc6a845591bd665bbc969.png)|Created
iacuniform_s[3]||![iacuniform_s[3]](http://images.aurorastation.org/images/27da83e54d05e4508a58513f098475a5.png)|Created
</details>
<details><summary>icons/obj/clothing/hats.dmi</summary>

Key | Old | New | Status
--- | --- | --- | ---
iac_helmet||iac_helmet|Created
iachat||iachat|Created
iachat_flipped||iachat_flipped|Created
</details>
<details><summary>icons/obj/clothing/masks.dmi</summary>

Key | Old | New | Status
--- | --- | --- | ---
iacmask||iacmask|Created
iacmask_r||iacmask_r|Created
</details>
<details><summary>icons/obj/clothing/suits.dmi</summary>

Key | Old | New | Status
--- | --- | --- | ---
iac_vest_s||iac_vest_s|Created
</details>
<details><summary>icons/obj/clothing/uniforms.dmi</summary>

Key | Old | New | Status
--- | --- | --- | ---
iacuniform||iacuniform|Created
</details>

File: code/modules/clothing/under/jobs/medsci.dm [hunk 1]
```diff
@@ -75,6 +75,17 @@
...
 	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 10, rad = 0)
 
+/obj/item/clothing/under/rank/iacjumpsuit
+	desc = "It's a and white jumpsuit, the IAC logo plastered across the back."
+	name = "IAC uniform"
+	icon_state = "iacuniform"
+	item_state = "iacuniform"
+	permeability_coefficient = 0.50
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 15, rad = 5)
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO
```
Comment (Alberyk) [inline code/modules/clothing/under/jobs/medsci.dm]: Why is this var needed?

File: code/modules/clothing/head/soft_caps.dm [hunk 2]
```diff
@@ -78,3 +78,9 @@
 	name = "corporate security cap"
 	desc = "It's field cap in corporate colors."
 	icon_state = "corpsoft"
+
+
+/obj/item/clothing/head/soft/iacberet
```
Comment (Alberyk) [inline code/modules/clothing/head/soft_caps.dm]: Why is this named beret but is a soft cap?

File: code/modules/clothing/under/jobs/medsci.dm (hunk 1 above)
Comment (dragoslava) [reply code/modules/clothing/under/jobs/medsci.dm]: which

Comment (Alberyk) [reply code/modules/clothing/under/jobs/medsci.dm]: body_parts_covered

**Pull request merged**